service/src/test/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiControllerAzureDatabaseTest.java

```diff
@@ -0,0 +1,142 @@
+package bio.terra.workspace.app.controller;
```

Comment:
I tried to be a good scout and add some unit test coverage for the main create and delete database operations. But I am very new to WSM so please lmk if something important is missing here.

Reply:
This seems like a reasonable amount of coverage for this change; fwiw `deleteDatabase` + `createAzureDatabaseWithNoParameters` NPE when I run locally so perhaps there is some mocking that needs cleaning up?

Reply:
Yeah, I trying to understand what I am missing in terms of mocks. I was naively following the examples for disks and VMs as a first pass